### ISSUE TITLE
Additional tests needed for .github workflow

### ISSUE THREAD

**Issue by vsoch** [urlstechie/urlchecker-action#43]
Currently, we do one run that does basic checks for the testing repository. We would want to add additional runs that:

 - use save and verify that the file exists
 - use cleanup to verify that the repository is deleted
 - others that might be useful

We absolutely don't want any PRs being merged that possibly break any previous functionality.

**Comment by SuperKogito** [comment]
yes, the coverage is quite low atm and I intend to work on improving it.
I can take on this if you don't mind.

**Comment by vsoch** [comment]
Yes that would be great! And don't forget to do feature branch and PR, and I'll be sure to give a speedy review :)

**Comment by vsoch** [comment]
@SuperKogito we need to prioritize this issue, over any changes to urlchecker-python. We need to make sure that the action is working for multiple different use cases, and we don't run into issues like we did with the cleanup arg!

**Comment by SuperKogito** [comment]
I agree, I am working on a PR for the test-repo that hopefully will solve this, but testing the changes might become messy because I want to trigger the tests following the action releases. It would be better to do it without a release but I am not sure if that is even possible. If you have better ideas on how to test the action, I am all ears :)
